### PR TITLE
Add new js-tiny-world task

### PR DESCRIPTION
**Pull request by okkkko**
_Have your app published on [GitHub pages](https://github.com/kottans/frontend-2021-homeworks/blob/main/publish-your-app.md). Make sure your PR title contains the task name. Change task name and links below as appropriate. Remove this paragraph._

### A Tiny JS World

[Demo](https://okkkko.github.io/a-tiny-JS-world/)
[Code base](https://github.com/okkkko/frontend-2021-homeworks/tree/js_tiny_world/submissions/okkkko/a_tiny_js_world)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by OleksiyRudenko** [comment]
The code base is expected to lead to your app repo on GitHub.
Please pay more attention to instructions.

image

File: submissions/okkkko/a_tiny_js_world/index.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	species:"cat-woman",
+	name:"Selina",
+	gender:"female",
+	legs:2,
+	hands:2,
+	saying:cat.saying,
+	friends: ["Milka"," Lisa"]
+}
+const inhabitants = [dog, cat, man, woman, catWoman];
+const properties = ["species", "name", "gender", "legs", "hands", "saying", "friends"]
```
Comment (OleksiyRudenko) [inline submissions/okkkko/a_tiny_js_world/index.js:57]: JS allows semicolons ommisiion. Both semicoloned and semicolonless styles are legal. But stick to either consistently.

File: submissions/okkkko/a_tiny_js_world/index.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+const formString = inhabitant => {
+	return properties.map(prop=>inhabitant[prop]).join("; ")
+}
```
Comment (OleksiyRudenko) [inline submissions/okkkko/a_tiny_js_world/index.js:60]: If `return` is the only statement in an arrow function the notation can be shorter.
`const fn = argument => { return something; };` is equivalent to `const fn = argument => something;`

**Review by OleksiyRudenko — APPROVED** [review]
@okkkko well done!
Approved with reservations. Check the comment below.

**Comment by okkkko** [comment]
> The code base is expected to lead to your app repo on GitHub.
> Please pay more attention to instructions.
> 
Sorry, I will be more attentive.

File: submissions/okkkko/a_tiny_js_world/index.js (hunk 2 above)
Comment (okkkko) [reply submissions/okkkko/a_tiny_js_world/index.js:60]: Understood, thanks for the review!

**Pull request merged**